### DISCUSSION TITLE
import a SCSS file as a CSS string using Parcel, SASS and TypeScript

### DISCUSSION THREAD

**Discussion by drachehavoc** [parcel-bundler/parcel#6333] [General]
Sorry for the noob question, but I'm new using parcel and a need someone to "talk"  :stuck_out_tongue:

I also make this same question in [StackOverflow](https://stackoverflow.com/questions/67645752/), but anyone give me attention :sob:

So, her is my doubt:

Is there a way to import SCSS as CSS using Parcel Bundler + SASS + TypeScript?

I have a SCSS file called `file.scss`
```SCSS
div {
    span: {
        background: red
    }
}
```

so I want to import it as a CSS string in TypeScript, I'm tryng something like this:
```ts
import cssString from "./file.scss"

console.log(cssString)
         // ^^^^^^^^^ Expected value => div span: { background: red } 
```

But isn't working properly.

So, i need to know, there is a way to do that?

**Comment by mischnic** [comment]
With Parcel 2, do
```js
import cssString from "bundle-text:./file.scss;"
```

https://v2.parceljs.org/configuration/plugin-configuration/#predefined-(offical)-named-pipelines

**Reply by whaaaley** [reply]
Is there a new place that this is described in? Link is 404.

**Reply by mischnic** [reply]
https://parceljs.org/features/bundle-inlining/#inlining-a-bundle-as-text
